plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/replay/ReplayFlowFactoryAction.java

```diff
@@ -52,7 +52,7 @@ class ReplayFlowFactoryAction extends InvisibleAction implements CpsFlowFactoryA
 
     private String replacementMainScript;
     private final Map<String,String> replacementLoadedScripts;
-    private transient final boolean sandbox;
+    private final boolean sandbox;
```

Comment:
Could avoid persisting it in `build.xml` as well, though unlike `replacementMainScript` the size impact is negligible.